Billing.API/Services/Invoice/InvoiceService.cs

```diff
@@ -16,10 +17,12 @@ public class InvoiceService : IInvoiceService
     {
         private readonly ILogger<InvoiceService> _logger;
         private readonly IOptions<InvoiceProviderOptions> _options;
+        private readonly CryptoHelper _cryptoHelper;
 
-        public InvoiceService(ILogger<InvoiceService> logger, IOptions<InvoiceProviderOptions> options)
+        public InvoiceService(ILogger<InvoiceService> logger, CryptoHelper cryptoHelper, IOptions<InvoiceProviderOptions> options)
```

Comment:
CryptoHelper no es necesariop aquí

```suggestion
        public InvoiceService(ILogger<InvoiceService> logger, IOptions<InvoiceProviderOptions> options)
```